scripts/package.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * This script builds the distributable packages. It assumes that we have _just_
+ * built the JavaScript parts.
+ */
+
+'use strict';
+
+import childProcess from 'child_process';
+import * as fs from 'fs/promises';
+import * as path from 'path';
+
+import buildInstaller from './lib/installer-win32';
+
+import { spawnFile } from '@pkg/utils/childProcess';
+
+/** Get the argument value (if any) for any of the given argument names */
+function getArgValue(args: string[], ...argNames: string[]): string | undefined {
```

Comment:
Since `getArgValue` is only called from one place, I think we can drop the `...` from `argNames` and require passing in an array of options. And `argNames` should be called something more like `optionNames`, because an arg can be either an option or an associated value.